lib/AppInfo/Application.php

```diff
@@ -68,5 +77,61 @@ public function register(IRegistrationContext $context): void {
 	}
 
 	public function boot(IBootContext $context): void {
+		$context->injectFn([$this, 'addPersonalNavigationEntries']);
```

Comment:
Im wondering if we can do this lazy in some way, maybe on a BeforeTemplateRenderedEvent so we don't trigger this logic on every request

Reply:
Yes, this disturbs me, too.